packages/mrgn-ui/src/components/action-box-v2/store/action-box-store.ts

```diff
@@ -7,6 +7,7 @@ interface ActionBoxState {
   isSettingsDialogOpen: boolean;
   priorityFee: number;
   slippageBps: number;
+  platformFeeBps: number;
```

Comment:
this is an oversight from me, this logic is missing everywhere in the actionbox. Good spot